frontend/main.py

```diff
@@ -252,6 +260,17 @@ def delete_correction(queue_name, correction_id, version):
     get_correction_table().delete_item(Key={'queue': queue_name, 'id': str(
         correction_id)}, ExpressionAttributeValues={':v1': version}, ConditionExpression='version = :v1')
 
+def update_correction_queue(queue_name, correction_id, version, updated_queue_name):
+    try:
+        #it is not possible to update an attribute that forms the Key of an item thus we need to copy/delete/put in DynamoDB.
+        correction = read_correction(queue_name, correction_id, version)
```

Comment:
let's read it outside the try and return not_found like skip_correction method does